packages/aws-cdk-lib/aws-stepfunctions/lib/activity.ts

```diff
@@ -222,4 +269,11 @@ export interface IActivity extends IResource {
    * @attribute
    */
   readonly activityName: string;
+
+  /**
+   * The symmetric customer managed KMS key for server-side encryption of the activity inputs.
```

Comment:
```suggestion
   * The symmetric customer-managed KMS key for server-side encryption of the activity inputs.
```
Please change any occurrence of `customer managed` to `customer-managed`.

Reply:
Should "customer-managed" be hyphenated? KMS and Step Functions docs don't use a hyphen

https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#customer-cmk
https://docs.aws.amazon.com/step-functions/latest/dg/encryption-at-rest.html

Reply:
Hmmm in that case we can leave it as `customer managed`. Grammatically it should contain the hyphen.

Reply:
Will omit using hyphen even though it's grammatically correct.